### DISCUSSION TITLE
Error when using select name "attributes"

### DISCUSSION THREAD

**Discussion by martijnburger** [remix-run/remix#1545] [Q&A] [answered]
Hey guys! I just spend two hours debugging a naming issue in the following code:

```
        <select name="attributes" defaultValue={data.selectedAttributes}>
          {data.attributes.map(attribute =>
            <option key={attribute.name} value={attribute.name}>{attribute.name} - {attribute.description}</option>
          )}
        </select>

```
The problem is that I can't seem to name the select box "attributes". The browser goes haywire in an error logging loop. Anyone know if this is caused by react or by remix? If I disable the scripts tag the problem resolves. Also if I change the name to i.e. "foo".

Here is the top part of the console log:
[error.txt](https://github.com/remix-run/remix/files/7896891/error.txt)

**Comment by sergiodxa** [comment, marked as answer]
It looks like a name conflict probably because attributes is used for something else in Js, that's why without scripts it work. There are a few field names like `name` or `action` which can have this issue.
